### PR TITLE
📖 Update CAPZ quickstart variables to include new AZURE_JSON_B64 secret

### PR DESCRIPTION
**Pull request by CecileRobertMichon**
**What this PR does / why we need it**: Following up on https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/653, we need to update the quickstart to include the new secret var.

/hold until CAPZ v0.4.6 release

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

File: docs/book/src/user/quick-start.md [hunk 1]
```diff
@@ -375,6 +375,27 @@ rm -f "${SSH_KEY_FILE}" 2>/dev/null
...
+    "vmType": "vmss",
+    "vnetName": "${AZURE_VNET_NAME}",
+    "vnetResourceGroup": "${CLUSTER_NAME}",
+    "subnetName": "${CLUSTER_NAME}-node-subnet",
+    "routeTableName": "${CLUSTER_NAME}-node-routetable",
+    "loadBalancerSku": "standard",
+    "maximumLoadBalancerRuleCount": 250,
+    "useManagedIdentityExtension": false,
+    "useInstanceMetadata": true
+}" | base64 | tr -d '\r\n')
```
[the remark this answers is not in the document]
Comment (nader-ziada) [reply docs/book/src/user/quick-start.md:398]: tested that and can confirm its the same

**Comment by CecileRobertMichon** [comment]
@vincepri not yet, holding until CAPZ release (with new templates)

**Pull request merged**